content/javascript/concepts/arrays/terms/push/push.md

```diff
@@ -12,42 +12,40 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
```

Comment:
Description needs to be edited to say "Adds one or more elements to the end of the array and returns the new length."